### PR TITLE
added Pie chart for exercise 2

### PR DESCRIPTION
**Pull request by mdshohelrana**

### CODE REVIEW

File: .github/workflows/hello-world.yml [hunk 1]
```diff
@@ -1,22 +1,38 @@
-name: Python Hello World Workflow
+name: Python Workflows
```
Comment (subhashbel) [inline .github/workflows/hello-world.yml:1]: Name will be GitHub Actions Workflows

File: generate_chart.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import matplotlib.pyplot as plt
+
+# Pie chart data
+labels = ['Apples', 'Bananas', 'Cherries', 'Dates']
+sizes = [15, 30, 45, 10]
+colors = ['#ff9999','#66b3ff','#99ff99','#ffcc99']
+explode = (0.1, 0, 0, 0)  # explode the 1st slice (Apples)
```
Comment (subhashbel) [inline generate_chart.py:7]: Add one fruit Mango.

**Review by subhashbel — CHANGES_REQUESTED** [review]

File: .github/workflows/hello-world.yml (hunk 1 above)
Comment (mdshohelrana) [reply .github/workflows/hello-world.yml]: Fixed

File: generate_chart.py (hunk 2 above)
Comment (mdshohelrana) [reply generate_chart.py]: Added

**Review by subhashbel — APPROVED** [review]
Looks good to me

**Pull request merged**